log.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/docker/docker/pkg/jsonmessage"
+	"github.com/docker/docker/pkg/term"
+)
+
+var cmdLog = &Command{
+	Run:      runLog,
+	Usage:    "log",
+	NeedsApp: true,
+	Category: "app",
+	Short:    "stream app log lines",
+	Long: `
+Log prints the streaming application log.
+   Examples:
+    $ emp log -a acme-inc
+    2013-10-17T00:17:35.066089+00:00 app[web.1]: Completed 302 Found in 0ms
+    ...
+`,
+}
+
+func runLog(cmd *Command, args []string) {
+	if len(args) != 0 {
+		cmd.PrintUsage()
+		os.Exit(2)
+	}
+
+	r, w := io.Pipe()
+	appName := mustApp()
+	endpoint := fmt.Sprintf("/apps/%s/log-sessions", appName)
+
+	go func() {
+		must(client.Post(w, endpoint, nil))
```

Comment:
Like I said in https://github.com/remind101/empire/pull/651#discussion_r38464676, if we just change empire to stream raw text, then we can just `must(client.Post(os.Stdout, endpoint, nil))`